insert.md

```diff
@@ -165,38 +165,127 @@ If you don't list column names, the statement will use the columns of the table
 ~~~
 
 ### Insert and Return Values
-
+  
+In this example, the `RETURNING` clause returns the `id` and `balance` of the rows inserted. The language-specific versions assume that you have installed the relevant [client drivers](install-client-drivers.html). 
+{{site.data.alerts.callout_success}}To mirror the behavior of MySQL's <code>LAST_INSERT_ID()</code> function, you would use <code>RETURNING</code> to return just the column containing the row ID.{{site.data.alerts.end}}
+
+<div id="step-three-filters" class="filters clearfix">
+    <button class="filter-button current" data-language="shell">Shell</button>
+    <button class="filter-button" data-language="python">Python</button>
+    <button class="filter-button" data-language="go">Go</button>
+</div>
+
+<div class="filter-content current" markdown="1" data-language="shell">
+<p></p>
 ~~~ sql
-> INSERT INTO accounts (id, balance) VALUES (DEFAULT, 5000.99) RETURNING id;
+> INSERT INTO accounts (id, balance) 
+  VALUES (1, 1000), (2, 250) 
+  RETURNING id, balance;
```

Comment:
This example doesn't make a lot of sense. You wouldn't use RETURNING to get back the same data you just put in; you'd use it for IDs that are generated on the server side. The old example was better.